### PR TITLE
Impl pocket tablist packet

### PR DESCRIPTION
**Pull request by caoli5288**

### CODE REVIEW

File: src/protocolsupport/protocol/packet/id/PEPacketId.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
 		public static final int PLAY_ENTITY_ANIMATION = 0x2C;
 		public static final int PLAY_RESPAWN = 0x3D;
 		public static final int PLAY_PLAYER_GAME_TYPE = 0x3E;
-
+		public static final int PLAY_PLAYER_INFO = 63;
```
Comment (Shevchik) [inline src/protocolsupport/protocol/packet/id/PEPacketId.java]: hex

File: src/protocolsupport/protocol/packet/middleimpl/readable/play/v_pe/PlayerListItemPacket.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
+package protocolsupport.protocol.packet.middleimpl.readable.play.v_pe;
```
Comment (Shevchik) [inline src/protocolsupport/protocol/packet/middleimpl/readable/play/v_pe/PlayerListItemPacket.java]: tabs

File: src/protocolsupport/protocol/packet/middleimpl/readable/play/v_pe/PlayerListItemPacket.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        item.setAction(action.origin);
+        item.setItems(elementlist.stream().map(el -> {
+            PlayerListItem.Item i = new PlayerListItem.Item();
+            i.setUuid(el);
+            return i;
+        }).toArray(PlayerListItem.Item[]::new));
+        return item;
+    }
+
+    enum Action {
```
Comment (Shevchik) [inline src/protocolsupport/protocol/packet/middleimpl/readable/play/v_pe/PlayerListItemPacket.java]: protected

File: src/protocolsupport/protocol/packet/middleimpl/writeable/play/v_pe/PlayerListItemPacket.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        ByteBuf buf = cache.tabList().remove(origin);
+        if (!(buf == null)) {
+            return Collections.singletonList(buf);
+        }
+        switch (origin.getAction()) {
+            case ADD_PLAYER:
+            case UPDATE_GAMEMODE:
+            case UPDATE_LATENCY:
+            case UPDATE_DISPLAY_NAME:
+                return Collections.emptyList();
```
Comment (Shevchik) [inline src/protocolsupport/protocol/packet/middleimpl/writeable/play/v_pe/PlayerListItemPacket.java]: Some bungee plugins send their own tablist packets, so just returning nothing here won't work.

Comment (caoli5288) [reply src/protocolsupport/protocol/packet/middleimpl/writeable/play/v_pe/PlayerListItemPacket.java]: I think compatible with custom tablist plugin is not a good solution. pe's player list is much different with pc's tablist

Comment (caoli5288) [reply src/protocolsupport/protocol/packet/middleimpl/writeable/play/v_pe/PlayerListItemPacket.java]: I means the visual different. Plugins put lots of extra infos into tablist but pe's playerlist is better to only handle player name.

**Pull request closed without merging**